src/Ed25519VerificationKey2018.js

```diff
@@ -0,0 +1,459 @@
+/*!
+ * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
```

Comment:
No "use strict" for ESM files.

```suggestion
```

The above suggestion should remove both "use strict" and the whitespace line below it ... github seems to only show it removing the first line, not sure what will happen, jfyi.